Software-Write-Barrier.md

```diff
@@ -80,5 +81,13 @@ Other than the plugin, there's runtime flags to help check if there's missing ba
 - You can also additionally add -KeepRecyclerTrackData, which saves more tracking info to help the debugging.
 But to be noticed both -VerifyBarrierBit and -RecyclerVerifyMark can report false positives.
 
-If would like to test this with windows, run "jenkens\buildone.cmd x64 debug swb" to build the bits, the result will be under Build\VcBuild.SWB\
+If would like to test this with windows, run "jenkens\buildone.cmd x64 debug swb"
+to build the bits. Build binaries will be under Build\VcBuild.SWB\.
 
+E.g.,
+```
+jenkins\buildone.cmd x64 debug swb
+python test\runtests.py -d --swb --x64
+  --flags "-recyclerconcurrentstress -recyclerverifymark -verifybarrierbit -keeprecyclertrackdata -dumponcrash"
```

Comment:
The flags were briefly explained above already. Capturing background thread dump issue is complicated and may need further work (did not work for me last time, tons of time out), we'll address if run into it.